exocom-server/src/client-registry/client-registry.ls

```diff
@@ -38,19 +38,20 @@ class ClientRegistry
 
 
 
-  # Adds service routing configurations to the given setup
-  register-client: (service) ->
-    @clients[service.name] =
-      client-name: service.name
-      service-type: service.name
-      internal-namespace: @routing[service.name].internal-namespace
+  # registers the given service instance that just came online
+  register-client: (client) ->
+    @clients[client.name] =
+      client-name: client.name
```

Comment:
Yes, absolutely! Let's do that once we build support for multiple instances of roles. That feature needs proper feature specs, tests, etc, so I don't want to do that here in this PR.